Lib/test/support/bytecode_helper.py

```diff
@@ -103,7 +103,7 @@ def normalize_insts(self, insts):
             if isinstance(oparg, self.Label):
                 arg = oparg.value
             else:
-                arg = oparg if opcode in self.HAS_ARG else None
+                arg = oparg if opcode in self.HAS_ARG else 0
```

Comment:
Why? And why do some of the `'NOP'`s below in `test_peepholer.py` have a `None` operand and some have `0`?